### ISSUE TITLE
Alaska Bottom Trawl Survey Data: https://afsc-gap-products.github.io/gap_products/

### ISSUE THREAD

**Issue by EmilyMarkowitz-NOAA** [ioos/bio_data_guide#250]
### Contact details

emily.markowitz@noaa.gov

### Dataset Title

Alaska Bottom Trawl Survey Data

### Describe your dataset and any specific challenges or blockers you have or anticipate.

Over the last three years, NOAA Fisheries Alaska Fisheries Science Center (AFSC) Resource Assessment and Conservation Engineering (RACE) Division’s Groundfish Assessment Program conducts fisheries-independent bottom trawl surveys to monitor the condition of the demersal fish and crab stocks of Alaska. These data are developed to describe the temporal distribution and abundance of commercially and ecologically important groundfish species, examine the changes in the species composition of the fauna over time and space, and describe the physical environment of the groundfish habitat. The team has worked to develop new public-facing data platforms and data products for sharing survey catch and environmental findings. These open science and data-sharing initiatives aim to enhance transparency, facilitate efficient and reproducible research, and promote accountability. Learn more about our data product tables on our documentation website: https://afsc-gap-products.github.io/gap_products/content/akfin-metadata.html

### Info about "raw" Data Files.

We are sharing data products, not raw data, because of the nature of the data and needing to share the most fool-proof and intuitive version of the data. Raw data and specialized data products can be requested through our data request repository (https://github.com/afsc-gap-products/data-requests/issues).

**Comment by albenson-usgs** [comment]
Would this dataset be related to this one https://obis.org/dataset/0b7ba344-5ff5-4a72-8e20-43eb86b097da?

**Comment by EmilyMarkowitz-NOAA** [comment]
Good question but nope, that is fisheries-dependent (observer-collected) data and we collect fisheries-independent (survey-based) data. Thanks for checking!

**Comment by albenson-usgs** [comment]
Makes sense! I was excited to potentially see an update to that one above. If you happen to know who to talk to in order to update that dataset (which only has data to 2004) that would be fantastic :-)

**Comment by EmilyMarkowitz-NOAA** [comment]
Oooo not sure. Let me first direct you to @CraigFaunce-NOAA and he can't help, then they'll connect you with the right person!

**Comment by CraigFaunce-NOAA** [comment]
That looks like a version we shared with an outside user. I have a query in the works to get more info.

**Comment by CraigFaunce-NOAA** [comment]
@albenson-usgs, the same data may exist here - https://apps-afsc.fisheries.noaa.gov/ords/f?p=711

**Comment by albenson-usgs** [comment]
Fantastic! Thank you!

**Comment by CraigFaunce-NOAA** [comment]
> Fantastic! Thank you!

If that doesnt work, you can write michael.a.moon@noaa.gov
